src/Chapter1.hs

```diff
@@ -632,7 +650,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Integral a => a -> a
+sumLast2 n = (+) l s
```

Comment:
Any reason to prefer prefix operator notation here? :thinking: 
```suggestion
sumLast2 n = l + s
```

Reply:
No reason! I shall update.